react/utils/date/index.test.ts

```diff
@@ -0,0 +1,95 @@
+import { getNextDay, getNextTime, getPreviousTime } from './index'
+
+describe('getNextDay', () => {
+  it('returns the day after a given day', () => {
+    const input = new Date('2019-04-01T12:07:21.000-03:00')
+
+    const expectedOutput = new Date('2019-04-02T00:00:00.000-03:00')
+
+    expect(getNextDay(input)).toEqual(expectedOutput)
+  })
+})
+
+describe('getNextTime', () => {
+  it('works with minutes = 0', () => {
+    const input = new Date('2019-04-01T12:00:00.000-03:00')
+
+    const expectedOutput = new Date('2019-04-01T12:30:00.000-03:00')
+
+    expect(getNextTime(input)).toEqual(expectedOutput)
+  })
+
+  it('works with 0 < minutes < 30', () => {
+    const input = new Date('2019-04-01T12:23:16.000-03:00')
+
+    const expectedOutput = new Date('2019-04-01T12:30:00.000-03:00')
+
+    expect(getNextTime(input)).toEqual(expectedOutput)
+  })
+
+  it('works with minutes = 30', () => {
+    const input = new Date('2019-04-01T15:30:00.000-03:00')
+
+    const expectedOutput = new Date('2019-04-01T16:00:00.000-03:00')
+
+    expect(getNextTime(input)).toEqual(expectedOutput)
+  })
+
+  it('works with 0 < minutes <= 59', () => {
```

Comment:
`30 < minutes <= 59`?

Reply:
10:35, 14:42: 11:50, etc.
Is it unintuitive?

Reply:
No, but you wrote `0` instead of `30` in the test title hehe

Reply:
Oops, nice catch!